DotaStats/Modules/SearchPlayerModule/View/SearchPlayerTableViewCell.swift

```diff
@@ -20,11 +20,11 @@ final class SearchPlayerTableViewCell: UITableViewCell {
     }()
 
     private lazy var timeMatch: UILabel = {
-        let raiting = UILabel()
-        raiting.font = .systemFont(ofSize: 17)
-        raiting.textColor = ColorPalette.mainText
-        raiting.translatesAutoresizingMaskIntoConstraints = false
-        return raiting
+        let rating = UILabel()
```

Comment:
Лучше называть подобные переменные одинаково, например так
```suggestion
        let view = UILabel()
```
Это позволит не запариваться над именами и легко копи-пастить одинаковый код по настройке вьюх